Base/QTGUI/qSlicerModulesMenu.cxx

```diff
@@ -69,10 +69,11 @@ qSlicerModulesMenuPrivate::qSlicerModulesMenuPrivate(qSlicerModulesMenu& object)
   this->NoModuleAction = nullptr;
   this->DuplicateActions = false;
   this->ShowHiddenModules = false;
-  this->TopLevelCategoryOrder << "Wizards" << "Informatics" << "Registration"
-    << "Segmentation" << "Quantification" << "Diffusion" << "IGT"
-    << "Filtering" << "Surface Models" << "Converters" << "Endoscopy"
-    << "Utilities" << "Developer Tools" << "Legacy" << "Testing";
+  this->TopLevelCategoryOrder << qSlicerModulesMenu::tr("Wizards") << qSlicerModulesMenu::tr("Informatics") << qSlicerModulesMenu::tr("Registration")
+    << qSlicerModulesMenu::tr("Segmentation") << qSlicerModulesMenu::tr("Quantification") << qSlicerModulesMenu::tr("Diffusion")
```

Comment:
We use the abstract module factory class name as context for module category everywhere. If we find that it is hard to enforce this then we may reconsider, but for now let's try to follow this convention.